lib/analyzer/applications/php.ts

```diff
@@ -0,0 +1,99 @@
+import * as lockFileParser from "@snyk/composer-lockfile-parser";
+import { legacy } from "@snyk/dep-graph";
+import * as path from "path";
+import { DepGraphFact, TestedFilesFact } from "../../facts";
+
+import { DepTreeDep } from "../../types";
+import { AppDepsScanResultWithoutTarget, FilePathToContent } from "./types";
+
+interface ManifestLockPathPair {
+  manifest: string;
+  lock: string;
+}
+const PACKAGE_MANAGER_TYPE = "composer";
+
+export async function phpFilesToScannedProjects(
+  filePathToContent: FilePathToContent,
+): Promise<AppDepsScanResultWithoutTarget[]> {
+  const scanResults: AppDepsScanResultWithoutTarget[] = [];
+
+  const filePairs = findManifestLockPairsInSameDirectory(filePathToContent);
+
+  const shouldIncludeDevDependencies = false;
```

Comment:
Same question as above - would be want to have this defined somewhere where it'll be easier to find/change?
I hope it doesn't sound like I'm trying to be annoying :)

Reply:
I don't expect it to change often if at all, so I think this way is easier to maintain than an env variable.

Reply:
also this file was heavily, um, inspired by the [node](https://github.com/snyk/snyk-docker-plugin/blob/main/lib/analyzer/applications/node.ts#L31) implementation